blog/incident-report-week3/index.qmd

```diff
@@ -0,0 +1,30 @@
+---
+author: Mordred Boulais
+title: Incident Report Regarding First Professional Precis
+date: '2023-9-11'
+date-format: YYYY
+categories: [post, developer development, incident report, professional precis, SE1]
+description: <em>A discussion of what we struggled with and how to improve from our first incident.</em>
+---
+
+The overarching struggle with getting out our first blog post regarding `Software
+Engineering At Google` mostly centered on the team's lack of proper communication
+and mutual understanding - of Github and of the task at hand. This was amplified
+by a lack of secure and consistent access to WiFi, and of a universal
+understanding of the terms used to describe the task.
```

Comment:
Overall, this paragraph does not make clear "what went wrong". Can you clearly explain what went wrong at the outset?